### PR TITLE
MODBULKOPS-204 - Item record's column names cleanup

### PR DESCRIPTION
**Pull request by AntonAntonich**
[MODBULKOPS-204](https://folio-org.atlassian.net/browse/MODBULKOPS-204)

## Purpose
 
Currently the list of column names for item records is ordered almost randomly and it includes properties that are important for backend processes but are normally not exposed in the inventory UI.
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/bulkops/service/ElectronicAccessService.java [hunk 1]
```diff
@@ -28,17 +29,33 @@ public class ElectronicAccessService {
...
   public String electronicAccessToString(ElectronicAccess access) {
-    return String.join(SPECIAL_ARRAY_DELIMITER,
+    return electronicAccessToString(access, SPECIAL_ARRAY_DELIMITER);
+  }
+
+  public String itemElectronicAccessToString(ElectronicAccess access) {
+    return electronicAccessToString(access, ARRAY_DELIMITER);
+  }
+
+  public ElectronicAccess restoreElectronicAccessItem(String s) {
```
Comment (khandramai) [inline src/main/java/org/folio/bulkops/service/ElectronicAccessService.java:39]: Please, rename argument s to smth readable, at least - value.

File: src/main/java/org/folio/bulkops/service/ElectronicAccessService.java [hunk 2]
```diff
@@ -28,17 +29,33 @@ public class ElectronicAccessService {
...
+    return String.join(delimiter,
       isEmpty(access.getRelationshipId()) ? EMPTY : electronicAccessReferenceService.getRelationshipNameById(access.getRelationshipId()),
       isNull(access.getUri()) ? EMPTY : access.getUri(),
       isEmpty(access.getLinkText()) ? EMPTY : access.getLinkText(),
       isEmpty(access.getMaterialsSpecification()) ? EMPTY : access.getMaterialsSpecification(),
       isEmpty(access.getPublicNote()) ? EMPTY : access.getPublicNote());
   }
 
-  public ElectronicAccess restoreElectronicAccessItem(String s) {
+  private ElectronicAccess restoreElectronicAccessItem(String s, String delimiter) {
```
Comment (khandramai) [inline src/main/java/org/folio/bulkops/service/ElectronicAccessService.java:56]: Please, rename argument s to smth readable, at least - value.

Comment (AntonAntonich) [reply src/main/java/org/folio/bulkops/service/ElectronicAccessService.java:56]: Such naming was in original method .. ok ... rename previous arguments also  or only my method ?

Comment (khandramai) [reply src/main/java/org/folio/bulkops/service/ElectronicAccessService.java:56]: Just rename argument s to make code more readable. Thank you!

**Review by khandramai — APPROVED** [review]

**Review by siarhei-charniak — APPROVED** [review]

**Pull request merged**